### PR TITLE
Get a brief string summary of a contact map

### PR DESCRIPTION
**Pull request by marrts**
This can be useful for motion planning debugging.

### CODE REVIEW

File: tesseract_collision/core/include/tesseract_collision/core/types.h [hunk 1]
```diff
@@ -290,6 +290,47 @@ class ContactResultMap
   bool operator==(const ContactResultMap& rhs) const;
   bool operator!=(const ContactResultMap& rhs) const;
 
+  /** @brief Get a brief summary of the most frequently colliding link pair
+   * @return A string stream containing the collision summary
+   */
+  std::stringstream getCollisionSummary() const
```
Comment (Levi-Armstrong) [inline tesseract_collision/core/include/tesseract_collision/core/types.h:296]: Why not just return a string?

**Review by Levi-Armstrong — CHANGES_REQUESTED** [review]

File: tesseract_collision/core/include/tesseract_collision/core/types.h (hunk 1 above)
Comment (Levi-Armstrong) [reply tesseract_collision/core/include/tesseract_collision/core/types.h:296]: @marrts I think we can just return a string here.

**Review by Levi-Armstrong — APPROVED** [review]

File: tesseract_collision/core/include/tesseract_collision/core/types.h [hunk 2]
```diff
@@ -290,6 +290,46 @@ class ContactResultMap
   bool operator==(const ContactResultMap& rhs) const;
   bool operator!=(const ContactResultMap& rhs) const;
 
+  /** @brief Get a brief summary of the most frequently colliding link pair
+   * @return A string containing the collision summary
+   */
+  std::string getCollisionSummary() const
```
Comment (Levi-Armstrong) [inline tesseract_collision/core/include/tesseract_collision/core/types.h:296]: I think we also just name this `getSummary()`.

Comment (marrts) [reply tesseract_collision/core/include/tesseract_collision/core/types.h:296]: Good point, I don't know what other summary you'd expect from a `ContactResultMap`

**Review by Levi-Armstrong — APPROVED** [review]

**Pull request merged**